themes/infection/layout/partials/toc.ejs

```diff
@@ -21,7 +21,7 @@
         </li>
     <% }) %>
     <li><h3>Posts</h3></li>
-    <% site.posts.sort('date', -1).each(function (p) { %>
+    <% site.posts.sort('date', -1).limit(5).each(function (p) { %>
```

Comment:
this is to limit the number of displayed posts to 5, since the list now is too long.

![image](https://user-images.githubusercontent.com/3725595/104655837-59d0b080-56cf-11eb-940c-4f17664f3f00.png)